concepts/pointers/introduction.md

```diff
@@ -0,0 +1,131 @@
+# Introduction
+
+Like many other languages, C++ has pointers.
+You already know `references` and pointers are similar, but think of them as a level closer to the inner workings of your computer.
+Pointers are variables that hold memory addresses.
+They are used to directly interact with memory, enabling dynamic memory allocation and efficient data manipulation in C++.
+
+If you're new to pointers, they can feel a little mysterious but once you get used to them, they're quite straight-forward.
+
+They're a crucial part of C++, so take some time to really understand them.
+The bare-bone version in this concept is also called _dumb pointer_ or _raw pointer_.
+With modern C++ there are also `smart pointers`, the basic type is not smart at all and you have to handle all the work manually.
+
+Before digging into the details, it's worth understanding the use of pointers.
+Pointers are a way to share memory with other parts of our program, which is useful for two major reasons:
+1. When you have large amounts of data, making copies to pass between functions is very inefficient.
+  By passing the memory location of where the data is stored instead, you can dramatically reduce the resource-footprint of our programs.
+2. By passing pointers between functions, you can access and modify the single copy of the data directly, meaning that any changes made by one function are immediately visible to other parts of the program when the function ends.
+
```

Comment:
Points 1 and 2 can also be achieved through references. You could add two more bullet points that explain that pointers can be reassigned to point to a different object, and they can be a null-pointer, i.e. not pointing to any object.